examples/periodic_snapshotter.cpp

```diff
@@ -50,59 +60,95 @@
 namespace laser_assembler
 {
 
-class PeriodicSnapshotter
+class PeriodicSnapshotter: public rclcpp::Node
 {
 
 public:
 
-  PeriodicSnapshotter()
+  explicit PeriodicSnapshotter(const std::string & service_name)
+  : Node(service_name)
   {
-    // Create a publisher for the clouds that we assemble
-    pub_ = n_.advertise<sensor_msgs::PointCloud> ("assembled_cloud", 1);
-
-    // Create the service client for calling the assembler
-    client_ = n_.serviceClient<AssembleScans>("assemble_scans");
+    std::cerr<< " Inside PeriodicSnapshotter constructor " << "\n";
```

Comment:
This seems like debugging output that should be removed.

Reply:
Removed